### PR TITLE
Added admin Event type crud

### PR DESCRIPTION
**Pull request by HeahDude**

### CODE REVIEW

File: src/AppBundle/Entity/EventCategory.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * @UniqueEntity("name")
+ */
+class EventCategory
+{
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer", options={"unsigned": true})
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     */
+    protected $id;
```
Comment (hhamon) [inline src/AppBundle/Entity/EventCategory.php]: `private` ?

File: src/AppBundle/Admin/EventCategoryAdmin.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+class EventCategoryAdmin extends AbstractAdmin
+{
+    protected $datagridValues = [
+        '_page' => 1,
+        '_per_page' => 32,
+        '_sort_order' => 'ASC',
+        '_sort_by' => 'name',
+    ];
+
+    protected function configureShowFields(ShowMapper $showMapper)
```
Comment (mykiwi) [inline src/AppBundle/Admin/EventCategoryAdmin.php]: `: void`

File: src/AppBundle/Admin/EventCategoryAdmin.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    {
+        $showMapper
+            ->add('name', null, [
+                'label' => 'Nom',
+            ])
+        ;
+    }
+
+    protected function configureFormFields(FormMapper $formMapper)
```
Comment (mykiwi) [inline src/AppBundle/Admin/EventCategoryAdmin.php]: `: void`

File: src/AppBundle/Admin/EventCategoryAdmin.php [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    {
+        $formMapper
+            ->add('name', TextType::class, [
+                'label' => 'Nom',
+            ])
+        ;
+    }
+
+    protected function configureListFields(ListMapper $listMapper)
```
Comment (mykiwi) [inline src/AppBundle/Admin/EventCategoryAdmin.php]: `: void`

File: tests/AppBundle/Controller/ControllerTestTrait.php [hunk 5]
```diff
@@ -68,6 +69,11 @@ public function authenticateAsAdherent(Client $client, string $emailAddress, str
         return $client->followRedirect();
     }
 
+    private function getEventCategoryIdForName(string $categoryName): int
+    {
+        return (string) $this->manager->getRepository(EventCategory::class)->findOneBy(['name' => $categoryName])->getId();
```
Comment (mykiwi) [inline tests/AppBundle/Controller/ControllerTestTrait.php]: Why cast in string?

**Comment by hhamon** [comment]
What's the status of this PR?

**Comment by mykiwi** [comment]
It's ready to merge, but #1123 will be useless (like I said in the trello comment)

**Comment by tgalopin** [comment]
Let's merge then.

**Pull request merged**